source/server-selection/server-selection-tests.rst

```diff
@@ -281,7 +281,7 @@ Prose Test
 
 Multi-threaded and async drivers MUST also implement the following prose test:
 
-1. Configure a sharded cluster with two mongoses.
+1. Configure a sharded cluster with two mongoses. Use a 4.9.0 or newer server version.
```

Comment:
That particular feature of blockTimeMS was added in 4.9.0, but the option itself was added in 4.2.9 in [SERVER-41070](https://jira.mongodb.org/browse/SERVER-41070), so we can use that bound here.

Reply:
Oof, I misread the ticket. Updated.